apps/nextjs-website/src/messages/it.json

```diff
@@ -100,14 +100,16 @@
         "relatedLinksTitle": "Link utili",
         "speakersTitle": "Ospiti",
         "questionList": {
-            "createdAt": "Ora creazione",
-            "text": "Domanda",
+            "title": {
+                "highlightedQuestions": "Domande evidenziate",
+                "notHighlightedQuestions": "Domande"
+            },
             "hiddenBy": "Nascosta da",
```

Comment:
If you apply the suggestion in `WebinarQuestionRow.tsx` component above also this translation can be delete
```suggestion
```